app/services/public_interfaces/creator.rb

```diff
@@ -105,6 +105,8 @@ def metadata
           'service.beta.kubernetes.io/aws-load-balancer-ssl-cert' =>
             Configurations.interfaces.ssl_certificate,
           'service.beta.kubernetes.io/aws-load-balancer-backend-protocol' => 'http',
```

Comment:
Minor remark: what do you think of moving this line up as the first annotation? This way the remaining three SSL-related settings will be grouped together?

Reply:
> BTW, why is `.env` checked to this repo? In all my projects, I take extra steps to make sure that it's the first thing to get `.gitignore`d.

Me too.  I know not everyone follows this practice.  I prefer `sample.env` file in its place personally.  I don't want to change this since it is there possible use cases I am not aware of and its not the intended use case being applied.  Should I remove the value from `.env`?